benchmarks/hdg/CMakeLists.txt

```diff
@@ -166,14 +166,14 @@ endforeach() # dim
 ###################################################
 
 foreach(D RANGE 2 3 1)
-  foreach(P RANGE 0 4 1) #3
-    foreach(N RANGE 1 2 1)
-      foreach(E RANGE 2 4 1)
+  foreach(P RANGE 1 3 1) #3
+    foreach(N 1)#RANGE 1 2 1)
+      foreach(E 4)#RANGE 2 4 1)
         feelpp_add_application( cvg_mixedpoisson_${D}DP${P}G${N}E${E}
```

Comment:
@romainhild Could you replace all application 2DP2P1.. by only one and use the options case.dimension and case.discretization as the others toolboxes

Reply:
This is for the benchmarks, where I would like to keep the structure of the export repositories ( Dimension/PolyOrder/GeoOrder/ExprOrder ) because I use a script to compute the convergence graphs.

The applications in toolboxes are now using the `case.` options.

Reply:
@romainhild but you can give these options in your script. It avoids having a lot of executable

Reply:
Is there a way to change properly the repository depending on options ? 
Or should I just use `changeRepository` ? I thought that we should avoid using it.